src/components/MaintenanceBanner.js

```diff
@@ -63,6 +68,20 @@ export default class TestnetBanner extends React.Component {
       currentNetwork,
     );
 
+    if (!relevantMaintenance) {
+      if (currentNetwork === "test") {
+        return (
+          <div className="LaboratoryChrome__network_reset_alert s-alert">
+            <div className="so-chunk">
+              Failed to fetch testnet reset date. Your browser extension may be
+              blocking the call.
+            </div>
+          </div>
```

Comment:
Instead of displaying this error message, could we link to our status page where the details can be found out instead?

Reply:
Great idea!